_events/OSG-User-School-2022.md

```diff
@@ -0,0 +1,72 @@
+---
+short_title: OSG User School 2022 
+title: "OSG User School 2022, July 25-29"
+layout: events
+published: true
+
+excerpt: |
+    The OSG User School 2022 be held from Monday 25 July to Friday 29 July 2022
+    
+    Registration is now open for the OSG User School 2022 to be held July 25-29 in person at the University of Wisconsin-Madison campus.
+start_date: 2022-07-25
+end_date: 2022-07-29
+location: "University of Wisconsin-Madison"
+link: "https://opensciencegrid.org/user-school-2022/#applications"
+image: "/images/Madison_Skyline.jpg"
+header_image: "/images/Madison_Skyline.jpg"
+---
+
+{% capture main %}
+
+<p style="font-size: larger; font-weight: bold;">Applications for the OSG User School 2022 are now open!</p>
+
+Applications are now **open** for the OSG User School 2022! The School will occur on July 25–29, where you will learn to use high-throughput computing (HTC) systems to run large-scale computing applications at your campus or using the national-scale OSG.
```

Comment:
Easy mistake that I also made before told otherwise. The OSG operates the OSPool which you can run jobs on, but you cannot run jobs on the OSG which I like to think of as a group of people. 

```suggestion
Applications are now **open** for the OSG User School 2022! The School will occur on July 25–29, where you will learn to use high-throughput computing (HTC) systems to run large-scale computing applications at your campus or on the OSPool.
```